pkg/proxy/metaproxier/meta_proxier.go

```diff
@@ -152,7 +152,7 @@ func (proxier *metaProxier) OnEndpointSliceAdd(endpointSlice *discovery.Endpoint
 	case discovery.AddressTypeIPv6:
 		proxier.ipv6Proxier.OnEndpointSliceAdd(endpointSlice)
 	default:
-		klog.V(4).Infof("EndpointSlice address type not supported by kube-proxy: %s", endpointSlice.AddressType)
+		klog.ErrorS(nil, "EndpointSlice address type not supported", "type", endpointSlice.AddressType)
```

Comment:
I want to know why it is not InfoS but Errors?

Reply:
I changed it because of this comment https://github.com/kubernetes/kubernetes/pull/104928#discussion_r706561412

Reply:
Have changed it. PTAL🙈

Reply:
LGTM